frontend/ui/widget/multiinputdialog.lua

```diff
@@ -104,6 +104,12 @@ local MultiInputDialog = InputDialog:extend{
 function MultiInputDialog:init()
     -- init title and buttons in base class
     InputDialog.init(self)
+    -- Reset layout, we're not using InputDialog's own field
+    self.layout = {}
```

Comment:
Oh, good point, I forgot that buttons could be a thing.

Reply:
> it looks much better in practice on an actual 800x600 eink screen

still kinda rubbish though ;)

Reply:
On second thought: what buttons? :D. InputDialog never adds buttons to this anywhere? (Except for the weird case-sensitive thingy from... whatever is actually using that).

Reply:
I don’t really know how that worked before as it was clearly a complete mess, but it might be that they were being inherited, now that you clear that at the start they don’t show up anymore, obviously.

Reply:
Oh, those buttons, lol.

Whelp, no idea how they got there, but, yeah, fixed it ;p.

Reply:
I think the inheritance makes sense since the cycle went (field 1, buttons and then it would finally start with multi input fields)